src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -190,9 +190,14 @@ internal static void UpdateBackground(this EditText platformView, IView view)
 				mauiDrawable.Dispose();
 			}
 
-			var previousDrawable = platformView.Background;
+			if(platformView.Background is LayerDrawable layerDrawable)
+			{
+				platformView.Background = null;
+				layerDrawable.Dispose();
+			}
+
 			var backgroundDrawable = paint!.ToDrawable(platformView.Context);
-			LayerDrawable layer = new LayerDrawable(new Drawable[] { backgroundDrawable!, previousDrawable! });
+			LayerDrawable layer = new LayerDrawable(new Drawable[] { backgroundDrawable!, defaultBackground! });
```

Comment:
Can we pull this background from android resources? This same update method is called by all the picker handlers and the searchbox as well. If we can localize all of this to the extension method and just load this drawable from the android resources then this would all just work and it'd be theme change safe.

Reply:
I think we can pull the drawable for this from this

`ContextCompat.GetDrawable(Context, Resource.Drawable.abc_edit_text_material);`

https://stackoverflow.com/a/62015982

Reply:
Can we pull this background from android resources? This same update method is called by all the picker handlers and the searchbox as well. If we can localize all of this to the extension method and just load this drawable from the android resources then this would all just work and it'd be theme change safe.

I think we can pull the drawable for this from this

`ContextCompat.GetDrawable(Context, Resource.Drawable.abc_edit_text_material);`

https://stackoverflow.com/a/62015982